site/.vuepress/components/hero.vue

```diff
@@ -11,7 +11,11 @@
 			<div :class="['container', containerClass]">
 				<div class="hero-content">
 					<h3 class="hero-title">{{ title }}</h3><!-- /.hero-title -->
-
+					<div class="hero-subhead"
+						v-if="['Home'].includes($page.title)"
+			          >Solve the operational problem of deploying and 
+			            <br>managing distributed networks.
```

Comment:
I wonder if saying 'networks' as opposed to 'systems' or smth like makes it sound like it's primarily for Telecom use cases or it's just me who connects these two?

Reply:
Sounds like there is still some question on this? Anyone have thoughts?

Reply:
We kept networks for the two-pager document so it should be alright here too. Can double check on the call or if we forget then let's keep networks for now.